src/components/GameCard/GameCard.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import styles from './GameCard.module.css'
+import PropTypes from 'prop-types'
+
+const GameCard = ({ title, img }) => {
```

Comment:
The component should navigate to another component, it would be  applicable to add Router with Link to GameComponent.

Reply:
done, just need to add `<Router>` somewhere above that component